src/components/Forms.js

```diff
@@ -9,7 +9,8 @@ export const TextArea = props => {
     placeholder,
     value,
     imageItem,
-    icon
+    icon,
+    readOnly
```

Comment:
We should add or update our eslint to enforce trailing commas. I've gotten used to it at work at it makes for cleaner diffs. https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
I'm down for this, It's annoyed me more than once to copy/paste go and delete comma.